### ISSUE TITLE
Local files are deleted when setting up a new remote repository with GitHub

### ISSUE THREAD

**Issue by coderbot16** [GitJournal/GitJournal#217]
## Environment

**App version:** : 1.65.0+1631
**Device information:**
 - OS name and version: Android 10
 - Manufacturer: Google
 - Model: Pixel 3

## Description

**Expected behavior:** Local files are pushed to a new remote when it is set up

**Current behavior:** Local files are deleted without warning when setting up a new remote

## Steps to reproduce

1. Make some notes with no remote setup
2. Add a remote repository with GitHub, using the "automatic setup" approach
3. Once the new repository has been created, all current notes are deleted

## Others   

The debug log says "reference `refs/heads/master` not found" when trying to pull (since the github repository hasn't been initialized). Then, GitJournal does not push the local files, and they somehow end up being deleted.

**Comment by vHanda** [comment]
That's odd. Could you please attach the full debug logs? You can generate it by clicking on report a bug.

**Comment by coderbot16** [comment]
Sorry, but I have already reinstalled the app (so the logs are probably gone, unfortunately) and I wasn't able to reproduce the issue a second time. This was the very first time I had set up GitJournal with my GitHub account, and despite trying various orderings of setting up the repository (such as first creating it on GitHub while in the GitJournal UI) I couldn't reproduce it.

I'll close this issue for now, but if I ever end up reproducing it I'll reopen it.
